public/main.js

```diff
@@ -0,0 +1,31 @@
+function createAudioElement (blobURL) {
+  var audio = document.createElement('audio')
+  audio.setAttribute('src', blobURL)
+  audio.setAttribute('controls', 'controls')
+
+  return audio
+}
+
+navigator.mediaDevices.getUserMedia({ audio: true }).then(stream => {
+    var chunks = [];
+    var recorder = new MediaRecorder(stream)
+
+    recorder.ondataavailable = e => {
+      chunks.push(e.data)
+
+      if (recorder.state == 'inactive') {
+          const blob = new Blob(chunks, { type: 'audio/wav' })
+          const blobURL = URL.createObjectURL(blob)
+          const audioElement = createAudioElement(blobURL)
+          document.querySelector('#audioPlayback').appendChild(audioElement)
```

Comment:
Also, might want to clear the "old" blob by using `URL.revokeObjectURL(url)` in order to avoid memory leaks.